plugins/field-bitmap/src/field-bitmap.js

```diff
@@ -24,7 +24,12 @@ const EMPTY_PIXEL_COLOR = 'white';
  * Includes a grid of clickable pixels that's exported as a bitmap.
  */
 export class FieldBitmap extends Blockly.Field {
-  /** Constructor for the bitmap field. */
+  /**
+   *Constructor for the bitmap field.
+   * @param {Array.<Array<number>>} value 2D rectangular array of 1s and 0s
```

Comment:
Annotating types can be tricky! There's a few relevant rules here from the [Google JS style guide](https://google.github.io/styleguide/jsguide.html#jsdoc-nullability)
- Non-primitive types (like `Array` or `HTMLElement`) need to be marked with a `!` to denote that they aren't allowed to be null. Primitives (like string or number) don't need to be marked that way, so most of the rest of this file is fine, there's just the `Array`s and `Object`  that need to be marked.
- we don't use the `.` when specifying generics (you might see it in some of our old code but it shouldn't be in new code)
- Optional parameters get a `=` at the end. You can tell they are optional because they provide a default value in the constructor signature.

So overall this type would be `!Array<!Array<number>>=`

and the other parameters should be fixed up similarly